app/views/environments/welcome.html.erb

```diff
@@ -6,7 +6,7 @@
   <p>
     <%= _('If you are planning to use Foreman as an external node classifier you should provide information about one or more environments.') %>
     <%= _('This information is commonly imported from a pre-existing puppet configuration by the use of the') %>
-    <a href="http://www.theforeman.org/manuals/<%= SETTINGS[:version].short %>/index.html#4.2.2Classes" rel="external"><%= _('Puppet classes and environment importer') %></a>
+    <a href=foreman_doc_url(index.html#4.2.2Classes)" rel="external"><%= _('Puppet classes and environment importer') %></a>
```

Comment:
Either switch to `link_to` or use erb here, as this will output as `<a href="foreman_doc_url...`